README.md

```diff
@@ -75,54 +99,78 @@ The following code gives an example on the usage:
+```bibtex
+@article{quetschlich2025mqtpredictor,
+  title        = {{MQT Predictor: Automatic Device Selection with Device-Specific Circuit Compilation for Quantum Computing}},
+  author       = {Quetschlich, Nils and Burgholzer, Lukas and Wille, Robert},
+  year         = {2025},
+  journal      = {ACM Transactions on Quantum Computing (TQC)},
+  doi          = {10.1145/3673241},
+  eprint       = {2310.06889},
+  eprinttype   = {arxiv}
+}
+```
 
-## References
+### The Munich Quantum Toolkit (the project)
 
-In case you are using MQT Predictor in your work, we would be thankful if you referred to it by citing the following publication:
+When discussing the overall MQT project or its ecosystem, cite the MQT Handbook:
 
 ```bibtex
-@ARTICLE{quetschlich2025mqtpredictor,
-    AUTHOR      = {N. Quetschlich and L. Burgholzer and R. Wille},
-    TITLE       = {{MQT Predictor: Automatic Device Selection with Device-Specific Circuit Compilation for Quantum Computing}},
-    YEAR        = {2025},
-    JOURNAL     = {ACM Transactions on Quantum Computing (TQC)},
-    DOI         = {10.1145/3673241},
-    EPRINT      = {2310.06889},
-    EPRINTTYPE  = {arxiv},
+@inproceedings{mqt,
+  title        = {The {{MQT}} Handbook: {{A}} Summary of Design Automation Tools and Software for Quantum Computing},
+  shorttitle   = {{The MQT Handbook}},
+  author       = {Wille, Robert and Berent, Lucas and Forster, Tobias and Kunasaikaran, Jagatheesan and Mato, Kevin and Peham, Tom and Quetschlich, Nils and Rovara, Damian and Sander, Aaron and Schmid, Ludwig and Schoenberger, Daniel and Stade, Yannick and Burgholzer, Lukas},
+  year         = 2024,
+  booktitle    = {IEEE International Conference on Quantum Software (QSW)},
+  doi          = {10.1109/QSW62656.2024.00013},
+  eprint       = {2405.17543},
+  eprinttype   = {arxiv},
+  addendum     = {A live version of this document is available at \url{https://mqt.readthedocs.io}}
 }
 ```
 
 ---
 
 ## Acknowledgements
 
-This project received funding from the European Research Council (ERC) under the European Union's Horizon 2020 research
-and innovation program (grant agreement No. 101001318), was part of the Munich Quantum Valley, which is supported by the
-Bavarian state government with funds from the Hightech Agenda Bayern Plus, and has been supported by the BMWK on the
-basis of a decision by the German Bundestag through project QuaST, as well as by the BMK, BMDW, the State of Upper
-Austria in the frame of the COMET program, and the QuantumReady project within Quantum Austria (managed by the FFG).
+The Munich Quantum Toolkit has been supported by the European Research Council (ERC) under the European Union's Horizon 2020 research and innovation program (grant agreement No. 101001318), the Bavarian State Ministry for Science and Arts through the Distinguished Professorship Program, as well as the Munich Quantum Valley, which is supported by the Bavarian state government with funds from the Hightech Agenda Bayern Plus.
```

Comment:
Any reason this was changed to a single line? (Also in the other PRs)
Liked the version before because one didn't have to scroll as much.